storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import InputImage from "./InputImage";
+//fullRound, width, icon, placeholder
+describe("Render the InputImage component", () => {
```

Comment:
Test Cases failed at this component , please check it again

Reply:
Please take the last updated Branch, I have already updated  all my snapshot .

Reply:
I have updated all the snapshot files and upload all of them